openembedded-core-mbl/scripts/signctl/src/signctl.py

```diff
@@ -141,32 +152,35 @@ def parse_args():
     )
     dump_cmd.set_defaults(func=handle_gen_cmd)
 
-    sign_cmd = commands.add_parser("sign-tfa", help="Sign TF-A BL and FIP.")
-    sign_cmd.add_argument(
-        "--rpi-armstub8", type=abspath, help="Path to armstub8.bin"
+    sign_cmd = commands.add_parser(
+        "sign", help="Sign TF-A bootloader components and u-boot FIT binary."
```

Comment:
Interesting U-boot is in a FIT within the FIP?

Reply:
Comment changed in 139aff9. U-boot is in the FIP, but the FIT image containing kernel, dtb and initramfs isn't.